.npmignore

```diff
@@ -0,0 +1,12 @@
+scripts
```

Comment:
Why we have to create ``.npmignore`` file? As I know, ``.gitignore`` will be automatically used if ``.npmignore`` doesn't exist? Is this not good enough?